src/modules/ce/components/ReferralStepAssignee.tsx

```diff
@@ -1,33 +1,47 @@
 import Chip from '@mui/material/Chip';
 import Typography from '@mui/material/Typography';
 import React, { Fragment, useMemo } from 'react';
-import { CeReferralStepSummaryFieldsFragment } from '@/types/gqlTypes';
+import {
+  CeReferralStepStatus,
+  CeReferralStepSummaryFieldsFragment,
+} from '@/types/gqlTypes';
 
 interface Props {
   step: CeReferralStepSummaryFieldsFragment;
 }
 const ReferralStepAssignee: React.FC<Props> = ({ step }) => {
   const { swimlane, assignees = [] } = step;
 
+  const locked = step.status === CeReferralStepStatus.Unavailable;
+
+  const commonChipSx = useMemo(() => {
+    return {
+      borderRadius: 1,
```

Comment:
These styles are duplicative with styles specified in ReferralStepStatusChip, maybe we need a `CommonRectangularChip` or something? I opened a thread [on figma](https://www.figma.com/design/znYqGn21VZO5zF2LjpnUiL?node-id=5424-153399#1270285775) because I don't quite understand the visual distinction between the rounded vs rectangular chips. So maybe hold off until we can resolve that!

Reply:
Sounds good, I'm fine to hold off on introducing square chips until we discuss more with design.
![Screenshot 2025-05-23 at 3 15 09 PM](https://github.com/user-attachments/assets/bee6019e-6f62-48e4-a854-4de468bbfb45)